GoogleSignIn/Sources/Public/GoogleSignIn/GIDAuthentication.h

```diff
@@ -64,9 +64,10 @@ typedef void (^GIDAuthenticationAction)(GIDAuthentication *_Nullable authenticat
 /// Get a valid access token and a valid ID token, refreshing them first if they have expired or are
 /// about to expire.
 ///
-/// @param action A callback block that takes a `GIDAuthentication` or an error if the attempt to
-///     refresh tokens was unsuccessful.  The block will be called asynchronously on the main queue.
-- (void)doWithFreshTokens:(GIDAuthenticationAction)action;
+/// @param completion A completion block that takes a `GIDAuthentication` or an error if the attempt
+///     to refresh tokens was unsuccessful.  The block will be called asynchronously on the main
+///     queue.
+- (void)doWithFreshTokens:(GIDAuthenticationAction)completion;
```

Comment:
Let's rename `GIDAuthenticationAction` to `GIDAuthenticationCompletion` for consistency.

Reply:
This method will be moved to GIDGoogleUser. If we can agree on this change I will make the change in my PR.

Reply:
I'm adding it here. You can pull it it into your branch after I merge, which I'll do momentarily.